api/prometheus/v1/api.go

```diff
@@ -366,6 +369,13 @@ type MetricMetadata struct {
 	Unit   string            `json:"unit"`
 }
 
+// Metadata models the metadata of a metric.
+type Metadata struct {
```

Comment:
Initially, I tried to re-use the `MetricMetadata` struct above. However, it felt misleading to return a type that will _always_ have two empty fields (target, metric).